cosmoz-data-nav.js

```diff
@@ -506,11 +506,11 @@ class CosmozDataNav extends translatable(mixinBehaviors([IronResizableBehavior],
 
 		if (matches.length === 0) {
 			// eslint-disable-next-line no-console
-			console.warn('trying to replace an item that is not in the list', id, item);
+			console.warn('List item replacement failed, no matching idPath', this.idPath, 'with id', id, 'in the item list', items, 'to replace with item', item);
```

Comment:
This was wrong because `console.warn()` was called with multiple arguments ...

Reply:
> This was wrong because `console.warn()` was called with multiple arguments ...

Not sure that I understand here, `console.warn()` can take multiple arguments like `console.log` do?

Reply:
@dotpointer warn was called with multiple arguments, but the tests were asserting that it was called with a single argument.

Reply:
Aha, thanks.